[2024-12-19T17:26:29.057+0900] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_crawling_to_db.post_crawl manual__2024-12-19T08:24:56+00:00 [queued]>
[2024-12-19T17:26:29.064+0900] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_crawling_to_db.post_crawl manual__2024-12-19T08:24:56+00:00 [queued]>
[2024-12-19T17:26:29.064+0900] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2024-12-19T17:26:29.078+0900] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): post_crawl> on 2024-12-19 08:24:56+00:00
[2024-12-19T17:26:29.083+0900] {standard_task_runner.py:57} INFO - Started process 136 to run task
[2024-12-19T17:26:29.086+0900] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_crawling_to_db', 'post_crawl', 'manual__2024-12-19T08:24:56+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/dag_crawling_to_db.py', '--cfg-path', '/tmp/tmpchrb8z2o']
[2024-12-19T17:26:29.089+0900] {standard_task_runner.py:85} INFO - Job 4: Subtask post_crawl
[2024-12-19T17:26:29.130+0900] {task_command.py:415} INFO - Running <TaskInstance: dag_crawling_to_db.post_crawl manual__2024-12-19T08:24:56+00:00 [running]> on host 6430ede81e67
[2024-12-19T17:26:29.196+0900] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_crawling_to_db' AIRFLOW_CTX_TASK_ID='post_crawl' AIRFLOW_CTX_EXECUTION_DATE='2024-12-19T08:24:56+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-19T08:24:56+00:00'
[2024-12-19T17:26:29.197+0900] {logging_mixin.py:151} INFO - 점핏 시작 시간: 2024-12-19 17:26:29
[2024-12-19T17:26:29.321+0900] {logger.py:11} INFO - ====== WebDriver manager ======
[2024-12-19T17:26:29.397+0900] {logger.py:11} INFO - Get LATEST chromedriver version for google-chrome
[2024-12-19T17:26:29.475+0900] {logger.py:11} INFO - Get LATEST chromedriver version for google-chrome
[2024-12-19T17:26:29.546+0900] {logger.py:11} INFO - There is no [linux64] chromedriver "131.0.6778.204" for browser google-chrome "131.0.6778" in cache
[2024-12-19T17:26:29.547+0900] {logger.py:11} INFO - Get LATEST chromedriver version for google-chrome
[2024-12-19T17:26:29.838+0900] {logger.py:11} INFO - WebDriver version 131.0.6778.204 selected
[2024-12-19T17:26:29.841+0900] {logger.py:11} INFO - Modern chrome version https://storage.googleapis.com/chrome-for-testing-public/131.0.6778.204/linux64/chromedriver-linux64.zip
[2024-12-19T17:26:29.841+0900] {logger.py:11} INFO - About to download new driver from https://storage.googleapis.com/chrome-for-testing-public/131.0.6778.204/linux64/chromedriver-linux64.zip
[2024-12-19T17:26:29.981+0900] {logger.py:11} INFO - Driver downloading response is 200
[2024-12-19T17:26:30.456+0900] {logger.py:11} INFO - Get LATEST chromedriver version for google-chrome
[2024-12-19T17:26:30.655+0900] {logger.py:11} INFO - Driver has been saved in cache [/home/***/.wdm/drivers/chromedriver/linux64/131.0.6778.204]
[2024-12-19T17:26:31.010+0900] {logging_mixin.py:151} INFO - 점핏 종료 시간: 2024-12-19 17:26:31
[2024-12-19T17:26:41.016+0900] {logging_mixin.py:151} INFO - 원티드 시작 시간: 2024-12-19 17:26:41
[2024-12-19T17:26:41.027+0900] {credentials.py:1254} INFO - Found credentials in shared credentials file: /code/.aws/credentials
[2024-12-19T17:26:41.181+0900] {logger.py:11} INFO - ====== WebDriver manager ======
[2024-12-19T17:26:41.236+0900] {logger.py:11} INFO - Get LATEST chromedriver version for google-chrome
[2024-12-19T17:26:41.329+0900] {logger.py:11} INFO - Get LATEST chromedriver version for google-chrome
[2024-12-19T17:26:41.397+0900] {logger.py:11} INFO - Driver [/home/***/.wdm/drivers/chromedriver/linux64/131.0.6778.204/chromedriver-linux64/chromedriver] found in cache
[2024-12-19T17:26:41.641+0900] {logging_mixin.py:151} INFO - 원티드 종료 시간: 2024-12-19 17:26:41
[2024-12-19T17:26:41.642+0900] {logging_mixin.py:151} INFO - 인크루트 시작 시간: 2024-12-19 17:26:41
[2024-12-19T17:26:42.234+0900] {logger.py:11} INFO - ====== WebDriver manager ======
[2024-12-19T17:26:42.292+0900] {logger.py:11} INFO - Get LATEST chromedriver version for google-chrome
[2024-12-19T17:26:42.399+0900] {logger.py:11} INFO - Get LATEST chromedriver version for google-chrome
[2024-12-19T17:26:42.461+0900] {logger.py:11} INFO - Driver [/home/***/.wdm/drivers/chromedriver/linux64/131.0.6778.204/chromedriver-linux64/chromedriver] found in cache
[2024-12-19T17:26:42.824+0900] {logging_mixin.py:151} INFO - 인크루트 종료 시간: 2024-12-19 17:26:42
[2024-12-19T17:26:52.830+0900] {logging_mixin.py:151} INFO - 로켓펀치 시작 시간: 2024-12-19 17:26:52
[2024-12-19T17:26:52.912+0900] {logger.py:11} INFO - ====== WebDriver manager ======
[2024-12-19T17:26:52.966+0900] {logger.py:11} INFO - Get LATEST chromedriver version for google-chrome
[2024-12-19T17:26:53.032+0900] {logger.py:11} INFO - Get LATEST chromedriver version for google-chrome
[2024-12-19T17:26:53.098+0900] {logger.py:11} INFO - Driver [/home/***/.wdm/drivers/chromedriver/linux64/131.0.6778.204/chromedriver-linux64/chromedriver] found in cache
[2024-12-19T17:26:53.343+0900] {logging_mixin.py:151} INFO - 로켓펀치 종료 시간: 2024-12-19 17:26:53
[2024-12-19T17:27:03.349+0900] {logging_mixin.py:151} INFO - 잡코리아 시작 시간: 2024-12-19 17:27:03
[2024-12-19T17:27:03.459+0900] {taskinstance.py:1935} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/code/plugins/post_crawling.py", line 1988, in post_main
    jobkorea_crawling()
  File "/code/plugins/post_crawling.py", line 1185, in jobkorea_crawling
    db = pymysql.connect(
  File "/home/airflow/.local/lib/python3.10/site-packages/pymysql/connections.py", line 297, in __init__
    raise ValueError("port should be of type int")
ValueError: port should be of type int
[2024-12-19T17:27:03.467+0900] {taskinstance.py:1398} INFO - Marking task as UP_FOR_RETRY. dag_id=dag_crawling_to_db, task_id=post_crawl, execution_date=20241219T082456, start_date=20241219T082629, end_date=20241219T082703
[2024-12-19T17:27:03.480+0900] {standard_task_runner.py:104} ERROR - Failed to execute job 4 for task post_crawl (port should be of type int; 136)
[2024-12-19T17:27:03.528+0900] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-12-19T17:27:03.548+0900] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
